### PR TITLE
added changes for AKS migration

### PR DESCRIPTION
**Pull request by AnilDhawaleGitHub**
### Change description ###

AKS migration changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -16,5 +16,7 @@ def app = "api"
 withPipeline(type, product, app) {
     enableDockerBuild()
     installCharts()
+    enableAksStagingDeployment()
```
Comment (adusumillipraveen) [inline Jenkinsfile_CNP]: this is not needed till workshop as this will add some more steps to pipeline.

File: charts/bar-api/requirements.yaml [hunk 2]
```diff
@@ -2,8 +2,4 @@ dependencies:
   - name: java
     version: ~2.2.2
```
Comment (adusumillipraveen) [inline charts/bar-api/requirements.yaml]: Use latest java - 2.6.0

File: charts/bar-api/values.aat.template.yaml [hunk 3]
```diff
@@ -1,7 +1,10 @@
+postgresql:
+  enabled: true
```
Comment (adusumillipraveen) [inline charts/bar-api/values.aat.template.yaml]: No, postgresql pod shouldnt be used in aat. you should use existing PAAS DB

File: charts/bar-api/values.yaml [hunk 4]
```diff
@@ -1,31 +1,22 @@
...
-    init.sql: |-
-      CREATE USER bar WITH PASSWORD 'bar';
-      CREATE DATABASE bar WITH OWNER = bar ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
-  nameOverride: bar-db
-  persistence:
-    enabled: false
-
 java:
   # Don't modify below here
-  image: 'hmctspublic.azurecr.io/bar/api:latest'
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: Let latest image stay

File: charts/bar-api/values.yaml [hunk 5]
```diff
@@ -1,31 +1,22 @@
...
-  ingressHost: ${SERVICE_FQDN}
-  applicationPort: 8080
+   applicationPort: 8080
   keyVaults:
     "s2s":
       resourceGroup: rpe-service-auth-provider
       secrets:
         - microservicekey-bar-api
   environment:
-    SPRING_DATASOURCE_URL: "jdbc:postgresql://{{ .Release.Name }}-bar-db:5432/bar"
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: You still need this . Either here or in preview?

File: charts/bar-api/values.yaml [hunk 6]
```diff
@@ -1,31 +1,22 @@
...
     IDAM_CLIENT_BASE_URL: https://preprod-idamapi.reform.hmcts.net:3511
-    PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
-
     APPINSIGHTS_INSTRUMENTATIONKEY: fake-key
+    postgresql:
+      nameOverride: bar-db
+      postgresqlUsername: bar
+      postgresqlPassword: bar
+      postgresqlDatabase: bar
+      extraEnv:
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: You can get rid of this when using latest java chart

File: charts/bar-api/values.yaml [hunk 7]
```diff
@@ -21,11 +11,9 @@ java:
...
-
     SPRING_LIQUIBASE_ENABLED: true
-
-    S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     IDAM_CLIENT_BASE_URL: https://preprod-idamapi.reform.hmcts.net:3511
-    PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
-
     APPINSIGHTS_INSTRUMENTATIONKEY: fake-key
+    postgresql:
+      nameOverride: bar-db
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: i didnt mean u delete dn details

File: charts/bar-api/values.yaml [hunk 8]
```diff
@@ -21,11 +11,9 @@ java:
     SPRING_DATASOURCE_URL: "jdbc:postgresql://{{ .Release.Name }}-bar-db:5432/bar"
     SPRING_DATASOURCE_USERNAME: bar
```
Comment (adusumillipraveen) [inline charts/bar-api/values.yaml]: can u make these dynamic using {{Values}} notation ?

**Comment by AnilDhawaleGitHub** [comment]
closing as its not required

**Pull request closed without merging**